src/routes/controllers/roles.controller.js

```diff
@@ -0,0 +1,84 @@
+import models from '../../db/models';
+import {
+  successResponse,
+  errorResponse,
+  getUserbyEmail,
+} from '../../utils/helpers.utils';
+import { AUTHOR, ADMIN } from '../../utils/constants';
+
+const { User } = models;
+
+/**
+ * Fetch all users and roles
+ *
+ * @param {Object} req express request object
+ * @param {Object} res express response object
+ * @returns {*} success response
+ * @throws {*} error if database error
+ */
+export async function getAll(req, res) {
+  try {
+    const roles = await User.findAll({
+      attributes: ['id', 'email', 'username', 'role'],
+    });
+    return successResponse(res, 200, 'roles fetched successfully', roles);
+  } catch (error) {
+    return errorResponse(res, 500, 'database error', error.message);
+  }
+}
+
+/**
+ * Make a user an admin
+ *
+ * @param {Object} req express request object
+ * @param {Object} res express response object
+ * @returns {*} success response
+ * @throws {*} error if database error
+ */
+export async function makeAdmin(req, res) {
+  const { email } = req.body;
+  const role = ADMIN;
```

Comment:
we can remove this variable and use the admin constant instead.

Reply:
This has been resolved